src/main/resources/templates/project-detail.html

```diff
@@ -16,7 +16,28 @@ <h1 class="text-green-200 font-sarabun font-light font text-32" th:text="${title
         <img src="/img/miso_logo.svg" class="w-6" alt="View in MISO" title="View in MISO">
       </a>
     </div>
-    <div id="projectSummaryTableContainer"></div>
+
+    <div>
+      <div class="flex gap-2">
+        <form action="" method="get" id="afterDateForm"
+          class="font-inter font-medium text-12 text-black bg-grey-100 px-2 py-1 rounded-md hover:ring-2 ring-green-200 ring-offset-1 flex space-x-2 items-center">
+          <div>
+            <label for="date">After Date:</label>
+            <input type="date" id="afterDate" />
+          </div>
+        </form>
+        <form action="" method="get" id="beforeDateForm"
+          class="font-inter font-medium text-12 text-black bg-grey-100 px-2 py-1 rounded-md hover:ring-2 ring-green-200 ring-offset-1 flex space-x-2 items-center">
+          <div>
+            <label for="date">Before Date:</label>
+            <input type="date" id="beforeDate" value="" />
+          </div>
+        </form>
+        <div id="projectSummaryTableFilter"></div>
+      </div>
+    </div>
```

Comment:
There seems to be a lot of unnecessary structure here. Form submission isn't being used and change listeners could be set on the date pickers instead, so the form elements aren't useful. There are a few divs that don't seem to have a purpose either. I believe this could be simplified down to something like

```
<div (flex)>
  <label>
  <date picker>
  <label>
  <date picker>
</div>
```

and then the dropdown can be appended to the existing div